Chrome uses BoringSSL while CrOS is still on OpenSSL. OpenSSL doesn't have
<openss/mem.h> header file.

diff --git a/crypto/scoped_openssl_types.h b/crypto/scoped_openssl_types.h
index 73f7632..b392a07 100644
--- a/crypto/scoped_openssl_types.h
+++ b/crypto/scoped_openssl_types.h
@@ -11,7 +11,6 @@
 #include <openssl/ec.h>
 #include <openssl/ecdsa.h>
 #include <openssl/evp.h>
-#include <openssl/mem.h>
 #include <openssl/rsa.h>
 
 #include "base/memory/scoped_ptr.h"
